README.md

```diff
@@ -19,31 +19,22 @@ This repository is the **reference implementation** of
 It is written in Python and intended to conform to version 1.0 of the
 [TUF specification](https://theupdateframework.github.io/specification/latest/).
```

Comment:
The link to 1.0.0-ANNOUNCEMENT.md also needs to change at the top of this document.